proto/iotics/api/search.proto

```diff
@@ -33,6 +33,15 @@ service SearchAPI {
 
   // Receive all search responses associated to a set of Search request for a given client application ID.
   rpc ReceiveAllSearchResponses(SubscriptionHeaders) returns (stream SearchResponse) {}
+
+  // TODO - this will be supported in future but is out of scope for initial requirements
```

Comment:
Q: Do raw-RDF (as explorer query) and formatted (like classic search) outputs need to be supported immediately?